src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -127,6 +127,8 @@ private static bool GetLinqExpressionsBuiltWithIsInterpretingOnly()
 
         public static bool IsLineNumbersSupported => !IsNativeAot;
 
+        public static bool IsPinnedGCHandleBlittabilityEnforced => !IsNativeAot;
```

Comment:
We only enforce that the type is unmanaged (ie does not contain pointers) now.

This should be fixed in https://github.com/dotnet/runtime/blob/983c8f239a98812498d874ed36b7001bd764fdfe/src/coreclr/nativeaot/System.Private.CoreLib/src/System/Threading/Win32ThreadPoolNativeOverlapped.cs#L126-L155 instead of adding this.